megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -888,6 +936,42 @@ protected void okAction() {
             clientgui.updateButtonPanel(GamePhase.TARGETING);
         }
 
+        // unit display non tabbed
+        if (!GUIPreferences.getInstance().getDisplayStartTabbed()) {
+            boolean unitDisplayNonTabbedChanged = false;
+            int s = unitDisplayNonTabbed.getSize();
+
+            if ((s > 0) && (!unitDisplayNonTabbed.get(0).equals(UnitDisplayOrderPreferences.getInstance().getString("NonTabbedA1")))) {
```

Comment:
I'm seeing a lot of repeated magic numbers (s > 0, .get(0)) and magic strings (e.g. "NonTabbedA1") here, can we put them into enums or constants instead? It'll make this easier to maintain going forward.